src/lib/types/Events.ts

```diff
@@ -195,6 +196,13 @@ export const Events = {
 	 */
 	CommandApplicationCommandRegistryError: 'commandApplicationCommandRegistryError' as const,
 
+	// Registry finished
+	/**
+	 * Emitted once the application command registries have been initialized.
+	 * @param {Map<string, ApplicationCommandRegistry>} registries The initalised registries
+	 */
+	CommandApplicationCommandRegistryFinished: 'commandApplicationCommadnRegistryFinished' as const,
```

Comment:
I'd personally drop the Command prefix, but I'd like @favna 's thoughts too

Also small typo!
```suggestion
	CommandApplicationCommandRegistryFinished: 'commandApplicationCommandRegistryFinished' as const,
```